spec/controllers/hyrax/admin/admin_sets_controller_spec.rb

```diff
@@ -113,16 +113,11 @@
 
     describe "#update" do
       let(:admin_set) { create(:admin_set, edit_users: [user]) }
-      let(:permission_template) { Hyrax::PermissionTemplate.find_or_create_by(admin_set_id: admin_set.id) }
       it 'updates a record' do
-        # Prevent a save which causes Fedora to complain it doesn't know the referenced node.
-        expect_any_instance_of(AdminSet).to receive(:save).and_return(true)
         patch :update, params: { id: admin_set,
-                                 admin_set: { title: "Improved title", thumbnail_id: "mw22v559x", workflow_name: "one_step_mediated_deposit" } }
+                                 admin_set: { title: "Improved title" } }
```

Comment:
I'm not aware of the UI interaction. Please explain the removal of the workflow_name from the update.